tests/test_metric_initialization.py

```diff
@@ -0,0 +1,94 @@
+from abc import ABC, abstractmethod
```

Comment:
This is failing on Python 2.7...
https://github.com/rycus86/prometheus_flask_exporter/actions/runs/3428976773/jobs/5726562202

With 2.7 not super (or super not) supported anymore, I'm not too fussed, maybe wrap it in a try-except and raise a `SkipTest` when fails, or explicitly test for Python version <3 and have it skipped then.

Reply:
And just drop python version 2?🤔

Reply:
Are you asking it we should drop Python 2 support altogether?
Probably eventually yes, for now I'd keep it probably.

Reply:
Yes, I think it would be better to pull the plug.
Do you really want to enable people that want to make a web application with a language that hasn't had security updates in almost 3 years? 😅
Switching might be inconvenient but is there any good reason to keep supporting insecure and outdated versions?

Python version 3.5 and 3.6 are also [end-of-life](https://devguide.python.org/versions/)

Reply:
Yeah, I guess it has to happen at some point...
I can merge this tomorrow then, disable eol Python runners and cut a new release.